docs/site/data/downloads/0-9-1.yml

```diff
@@ -30,3 +30,5 @@ source:
     file_name: v0.9.1.zip
     checksum: ''
     size: ''
+ova:
+    - ova_page_link: 'https://customerconnect.vmware.com/downloads/details?downloadGroup=TCE-090&productId=1202&download=true&fileId=b1d59dca8908173f7441bb4d77d7e1d6&uuId=8283159f-927e-417d-9902-f26c5c0cf13d'
```

Comment:
Remove the leading `-` and dedent so this is just a property of the `ova` element.

Reply:
Done.